characterisation/exceptions/HO200116.test.ts

```diff
@@ -0,0 +1,47 @@
+import World from "../../utils/world"
+import { processPhase2Message } from "../helpers/processMessage"
+import { asnPath } from "../helpers/errorPaths"
+import generateMessage from "../helpers/generateMessage"
+
+jest.setTimeout(30000)
+
+describe.ifPhase2("HO200116", () => {
+  afterAll(async () => {
+    await new World({}).db.closeConnection()
+  })
+
+  it("creates a HO200116 exception when there are more than 100 offences (there are 110)", async () => {
+    const inputMessage = generateMessage("test-data/HO200116/Aho110Offences.xml.njk", {})
```

Comment:
In #598, we refactored the characterisation tests for Phase 2 so we don't need to create a input message XML for each test. Can we refactor these tests to remove the need of the XML files please?

See https://github.com/ministryofjustice/bichard7-next-tests/pull/600, for how previous tests have been refactored.

Let me know if you need support!